src/form/PMForm/CustomFormResponse.php

```
@@ -0,0 +1,77 @@
+		$this->checkExists($name);
+		return $this->data[$name];
+	}
+
+	public function getFloat(string $name) : float{
+		$this->checkExists($name);
+		return $this->data[$name];
+	}
+
+	public function getBool(string $name) : bool{
+		$this->checkExists($name);
+		return $this->data[$name];
```

PHPStan (2.1.15)

Method pocketmine\form\PMForm\CustomFormResponse::getBool() should return bool but returns mixed.


(return.type)